webpack.config.js

```diff
@@ -7,14 +7,15 @@ var ExtendedDefinePlugin = require('extended-define-webpack-plugin');
 var config = require(path.resolve(__dirname, 'app/utils/config.example.js'));
 
 var appRoot = path.resolve(__dirname, 'app/');
+var buildDir = path.resolve(__dirname, 'build');
 
 module.exports = {
   context: __dirname,
   entry: path.resolve(appRoot, 'init.js'),
   output: {
-    path: path.resolve(__dirname, 'build'),
+    path: buildDir,
     publicPath: '/dist/',
-    filename: 'bundle.js'
+    filename: 'bundle.[hash].js'
```

Comment:
getting

GET http://askdarcel.lvh.me:8080/dist/bundle.5d36568044b620a6bd28.js 404 (Not Found)